apps/web/src/pages/Applications/ApplicationProfilePage/components/DiversityEquityInclusion/DiversityEquityInclusion.tsx

```diff
@@ -0,0 +1,177 @@
+import React from "react";
+import { UserCircleIcon } from "@heroicons/react/24/outline";
+import { useIntl } from "react-intl";
+
+import { Heading, ToggleSection, Well } from "@gc-digital-talent/ui";
+
+import EquityOptions from "~/components/EmploymentEquity/EquityOptions";
+import { EquityKeys } from "~/components/EmploymentEquity/types";
+import { wrapAbbr } from "~/utils/nameUtils";
+import { hasAllEmptyFields } from "~/validators/profile/diversityEquityInclusion";
+
+import { SectionProps } from "../../types";
+import { getSectionIcon } from "../../utils";
+import SectionTrigger from "../SectionTrigger";
+import NullDisplay from "./NullDisplay";
+import Display from "./Display";
+
+const DiversityEquityInclusion = ({
+  user,
+  onUpdate,
+  isUpdating,
+}: SectionProps) => {
+  const intl = useIntl();
+  const [isEditing, setIsEditing] = React.useState<boolean>(false);
+  const isNull = hasAllEmptyFields(user);
+  const icon = getSectionIcon({
+    isEditing,
+    error: false,
+    completed: false,
+    fallback: UserCircleIcon,
+  });
+
+  const handleUpdate = (key: EquityKeys, value: unknown) => {
+    return onUpdate(user.id, {
+      [key]: value,
+    });
+  };
+
```

Comment:
the rest of the sections mention their name upon being updated, so this creates an odd one out if it just says "user updated"

![image](https://user-images.githubusercontent.com/40485260/232575157-6cc3daeb-82f6-4df4-97ae-40cbb28b78a9.png)

Reply:
Good catch, updated in [add more context to dei success message](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6296/commits/454c9ccaa16528ea7d0de7c1217a8922aad49320)